lms/templates/certificates/_accomplishment-footer.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
Will this cause any issues for URLs used on this page? (Will urls containing & be escaped to &amp;)?